src/bin/main.rs

```diff
@@ -194,52 +195,52 @@ fn real_main() -> Result<i32, std::io::Error> {
         .arg(Arg::with_name("margin").long("margin").multiple(true).takes_value(true).default_value("0,0,0,0"))
         .arg(Arg::with_name("min-height").long("min-height").multiple(true).takes_value(true).default_value("10"))
         .arg(Arg::with_name("height").long("height").multiple(true).takes_value(true).default_value("100%"))
-        .arg(Arg::with_name("no-height").long("no-height").multiple(true))
-        .arg(Arg::with_name("no-clear").long("no-clear").multiple(true))
-        .arg(Arg::with_name("no-clear-start").long("no-clear-start").multiple(true))
-        .arg(Arg::with_name("no-mouse").long("no-mouse").multiple(true))
+        .arg(Arg::with_name("no-height").long("no-height"))
```

Comment:
multiple() is deprecated, I feel like we should take the occasion to modify the other ones too

Reply:
It is fixed in #586